metadata.json

```diff
@@ -9,8 +9,9 @@
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 5.0.0"}
+    {"name":"puppet/tea","version_requirement":"= 0.2.1"}
```

Comment:
TEA is pre 1.0, so any API changes are allowed. The code we're depending on is not yet released, so this is just a dummy entry designed to fail.